tests/regression/test_bcs.py

```diff
@@ -1,403 +0,0 @@
-import pytest
```

Comment:
ok that was I mistake, I didn't noticed that `test_bcs.py` already existed. I'll put it back.